tests/test_filtsmooth/test_gaussfiltsmooth/filtsmooth_testcases.py

```diff
@@ -143,6 +143,7 @@ class LinearisedDiscreteTransitionTestCase(unittest.TestCase, NumpyAssertions):
 
     def test_transition_rv(self):
         """transition_rv() not possible for original model but for the linearised model"""
+        # pylint: disable=not-callable
```

Comment:
Why are these required?

Reply:
if I remember correctly, because the default values for `self.linearising_component` are `NotImplemented`, thus not callable